openhackathon-guacamole-auth-provider/pom.xml

```diff
@@ -62,6 +62,18 @@
             <version>1.0</version>
         </dependency>
 
+        <!-- Guice -->
+        <dependency>
+            <groupId>com.google.inject</groupId>

```

Comment:
Do we need include this dependency jar file in `/deploy/guacamole` just like the json jar file?